### ISSUE TITLE
2019-7-4

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#46]
981.model
982.1.4 - Examining test set performance
983.2 - Emojifier-V2: Using LSTMs in Keras:
984.2.1 - Overview of the model
985.2.2 Keras and mini-batching
986.2.3 - The Embedding layer
987.sentences_to_indexes
988.pretrain_embedding_layer
989.2.3 Building the Emojifier-V2
